kodilla-stream/src/test/java/com/kodilla/exception/homework/WarehouseTestSuite.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.exception.homework;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.function.BooleanSupplier;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WarehouseTestSuite {
+
+    @Test
+    public void testGetOrder_withException() {
+        //given
+        Warehouse warehouse = new Warehouse();
+        //when
+        //then
+        assertThrows(OrderDoesntExistException.class, () -> warehouse.getOrder("0"));
+    }
+    @Test
+    public void testGetOrder() throws OrderDoesntExistException {
+        // given
+        Warehouse warehouse = new Warehouse();
+        warehouse.addOrder(new Order("123"));
+        warehouse.addOrder(new Order("456"));
+        // when
+        Order result = warehouse.getOrder("123");
+        // then
+        assertEquals(result, warehouse.getOrder("123"));
```

Comment:
assertEquals(result.getNumber(), "123");
assertEquals(result, new Order("123"));  // jesli w klasie Order jest dobrze nadpisana metoda equals i hasCode, to to jest lepsze

int x = 5;
assertEquals(x, 5);